src/index.html

```diff
@@ -19,9 +19,423 @@
       rel="stylesheet"
       href="styles/index.scss"
```

Comment:
It seems like you are linking a SCSS file in your HTML. Browsers don't understand SCSS, they only understand CSS. You should compile your SCSS to CSS and then link the CSS file in your HTML.